pkg/identity/identity.go

```diff
@@ -527,3 +528,41 @@ func backupPath(path string) string {
 		pathExt,
 	)
 }
+
+// EncodePeerIdentity encodes the complete idenitity chain to bytes
+func EncodePeerIdentity(pi *PeerIdentity) []byte {
+	var chain []byte
+	chain = append(chain, pi.Leaf.Raw...)
+	chain = append(chain, pi.CA.Raw...)
+	for _, cert := range pi.RestChain {
+		chain = append(chain, cert.Raw...)
+	}
+	return chain
+}
+
+// DecodePeerIdentity Decodes the bytes into complete idenitity chain
```

Comment:
spelling of identity

Reply:
Extremely sorry overlooked the comment and checked ... will  fix it ..